duckduckgo/macos-browser
# Bump C-S-S to 4.39.0

Task/Issue URL: https://app.asana.com/0/0/1205693429987634/f
Tech Design URL:
CC:

**Description**:
Fixes an issue where strictHide element hiding rules aren't applied properly.

**Steps to test this PR**:
1. Visit https://www.first-party.site/features/element-hiding/, make sure all tests are green



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
